api/lib/notifications/notifications.js

```diff
@@ -157,7 +157,7 @@ function getPushLink(
 // Get relevant icon for notification
 // TODO: Upload icons to DO instead of passing relative links
 function getIcon({ eventType, data }) {
-  const notificationData = JSON.parse(data)
+  const notificationData = notification.properties || JSON.parse(data)
```

Comment:
I get here "notification is not defined"

Reply:
![image](https://user-images.githubusercontent.com/40721795/87356778-f8365a80-c562-11ea-91e0-9a3b281ee130.png)

Reply:
fixed